### ISSUE TITLE
Unpublished Helm chart

### ISSUE THREAD

**Issue by brettmorien** [indeedeng-alpha/harbor-container-webhook#3]
Hi folks. We are excited to try out this webhook, but are stuck by the lack of a Helm repository. Could you please create the helm repository? This is the most typical way using Github pages: https://medium.com/@mattiaperi/create-a-public-helm-chart-repository-with-github-pages-49b180dbb417

We have an example in a project of ours which publishes the index.yaml to a gh-pages branch if you don't have another way in mind.
https://github.com/orbit/orbit/tree/gh-pages
https://github.com/orbit/orbit/tree/master/.github

**Comment by brettmorien** [comment]
Updating to note this is less of a priority now once we converted to a Rancher catalog v2 which can use the Github repo directly. We are now blocked on the docker image being published to a private Dockerhub repo. We would like to try this webhook out if it works for your team, so I'm opening a separate issue to push to public Dockerhub.

**Comment by cnmcavoy** [comment]
I published the current release to [dockerhub](https://hub.docker.com/r/cnmcavoy/harbor-container-webhook), and added a barebones [chart repo](https://indeedeng-alpha.github.io/harbor-container-webhook).

**Comment by brettmorien** [comment]
Amazing. Thank you!
